scimath/units/tests/test_scalar_unit.py

```diff
@@ -61,18 +61,3 @@ def test_str(self):
         dimensionless_unit.label = "Cool unit"
         a = UnitScalar(1, units=dimensionless_unit)
         self.assertEqual(str(a), "UnitScalar (Cool unit): 1")
-
-
-def test_offset_unit_computations():
-    """ Executing some basic computations with a basic custom unit with offset.
-    """
-    my_u = unit(12, m.derivation, 14)
-    s1 = UnitScalar(3, units=my_u)
-    s2 = UnitScalar(5, units=my_u)
-    s3 = s1 + s2
-    assert_equal(s3, UnitScalar(8, units=my_u))
```

Comment:
this test already exists in this testcase - https://github.com/enthought/scimath/blob/ebee58f1df23085fa080f003e82e3d5d9aed2553/scimath/units/tests/test_scalar_unit.py#L23-L30